sklearn/sklearn_optuna_search_cv_simple.py

```diff
@@ -16,8 +16,8 @@
     clf = SVC(gamma="auto")
 
     param_distributions = {
-        "C": optuna.distributions.LogUniformDistribution(1e-10, 1e10),
-        "degree": optuna.distributions.IntUniformDistribution(1, 5),
+        "C": optuna.distributions.FloatDistribution(1e-10, 1e10),
```

Comment:
Could you use the `log` option of `FloatDistribution`, please?
https://optuna.readthedocs.io/en/latest/reference/generated/optuna.distributions.FloatDistribution.html#optuna.distributions.FloatDistribution.log

Reply:
Thank you for your comment. I have taken care of it.

Reply:
Great! Thanks.